hudi-utilities/pom.xml

```diff
@@ -361,10 +366,12 @@
     <dependency>
       <groupId>io.confluent</groupId>
       <artifactId>kafka-avro-serializer</artifactId>
+      <version>${confluent.version}</version>
```

Comment:
The version is already defined in the root POM so there is no need to do it in the `hudi-utilities`'s POM.